packages/web/src/components/molecules/DeleteModal/DeleteModal.spec.tsx

```diff
@@ -0,0 +1,107 @@
+import {fireEvent, render, waitFor} from '@testing-library/react';
+
+import DeleteModal from './DeleteModal';
+
+describe('DeleteModal', () => {
+  const onDelete = jest.fn();
+  const onClose = jest.fn();
+  const successMessage = 'Item deleted successfully';
+  const idToDelete = '123';
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders the title and subtitle', () => {
+    const {getByText} = render(
+      <DeleteModal
+        onDelete={onDelete}
+        onClose={onClose}
+        successMessage={successMessage}
+        idToDelete={idToDelete}
+        title="Delete Item"
+        subtitle="Are you sure you want to delete this item?"
+      />
+    );
+
+    expect(getByText('Delete Item')).toBeInTheDocument();
```

Comment:
You can use `getByTestId` and compare the text content. Current solution will work too, but I think that `test-id` is more specific. Same for other